### DISCUSSION TITLE
APIservice configuring incorrectly

### DISCUSSION THREAD

**Discussion by dmcstravick7** [kedacore/keda#4769] [Q&A / Need Help]
## Issue
- metrics-server APIservice is failing to come up (ServiceNotFound) `v1beta1.external.metrics.k8s.io`
- All Keda resources are deployed as `keda-operator*` in this case that includes `keda-operator-metrics-apiserver`. 
  - The spec for `/keda/config/metrics-server/api_service.yaml` contains spec.Service.Name for `keda-metrics-apiserver`.  **^ Issue**

## Fix
- Editing the APIservice and updating spec.Service.Name to `keda-operator-metrics-apiserver`.
- E.g 
  - ``` kubectl edit apiservice v1beta1.external.metrics.k8s.io ``` 
  - Change  spec.Service.Name from `keda-metrics-apiserver` --> `keda-operator-metrics-apiserver`.

## Steps to reproduce
 - Install Keda via helm chart 2.11.1 (note this has happened with previous versions)
 - Running any kubectl command such as `kubectl get pods` returns the below error (also returns pods)
 ```
57503 memcache.go:287] couldn't get resource list for external.metrics.k8s.io/v1beta1: the server is currently unable to handle the request
 ```

## Notes
- Could this be due to setting me not setting some value in the chart that I am unaware of?

**Comment by JorTurFer** [comment]
I lock this discussion, let's continue on the issue
https://github.com/kedacore/charts/issues/495
